experiments/AtmosLES/stable_bl_model.jl

```diff
@@ -148,14 +148,15 @@ function init_problem!(problem, bl, state, aux, localgeo, t)
         θ_liq = FT(265) + FT(0.01) * (z - z1)
     end
     θ = θ_liq
-    π_exner = FT(1) - _grav / (c_p * θ) * z # exner pressure
-    ρ = p0 / (R_gas * θ) * (π_exner)^(c_v / R_gas) # density
+    p = aux.ref_state.p
```

Comment:
@ilopezgp the other SBL cases are breaking, is it because our initialization is different?

Reply:
Yes, the error comes from using `density` here, which default the density to state.rho, which is not yet defined. I will push a fix.